### PR TITLE
Update BuildTools to 2.0.0-servicing-02026-05

### PR DESCRIPTION
**Pull request by eerhardt**
This version is produced by the release/2.0.0 branch of buildtools, which we will want to use for release/2.0.0 of source-build.

It also contains a fix that I need for core-setup.  See https://github.com/dotnet/buildtools/pull/1693

@dotnet/source-build-contrib

### CODE REVIEW

**Comment by weshaggard** [comment]
Do we need to update any patches?

**Comment by eerhardt** [comment]
> Do we need to update any patches?

I hope not.  We will see though.  My thinking is this buildtools version should "just work", since it is pretty close to the version we are currently using in buildtools.

**Comment by eerhardt** [comment]
Maybe it isn't as close as I first thought.  Looks like our current version was built on `21 June 2017` and the release/2.0.0 branch is more or less master from `3 Aug 2017`.  Hopefully no breaking changes came in between those dates.

**Comment by eerhardt** [comment]
Looks like I'll need to move the core-setup SHA forward so we have https://github.com/dotnet/core-setup/pull/3051.

**Comment by weshaggard** [comment]
looks reasonable to me.

**Comment by eerhardt** [comment]
test RHEL7.2 Release

**Comment by eerhardt** [comment]
test RHEL7.2 Release

**Comment by eerhardt** [comment]
test RHEL7.2 Release

**Comment by eerhardt** [comment]
test RHEL7.2 Release

**Comment by mmitche** [comment]
test RHEL7.2 Release (jenkins restarted/updated)

**Comment by eerhardt** [comment]
test RHEL7.2 Release

I'm getting a random error for not being able to download NuGet packages.  It seems to affect every other run.

File: targets/cli.props [hunk 1]
```diff
@@ -9,6 +9,8 @@
     <TarBallOutput>$(PackagesOutput)</TarBallOutput>
     <GitCommitCount>6908</GitCommitCount>
     <GitCommitHash>38031f6e5c3e21fa39d82aca300f5ed769be10a0</GitCommitHash>
+    <UpdateCommand>(set DOTNET_VERSION_URL=file:%2F%2F$(LocalBuildInfoRoot)) &amp; powershell.exe $(ProjectDirectory)build_projects/update-dependencies/update-dependencies.ps1 -Update</UpdateCommand>
+    <UpdateCommand Condition="'$(OS)' != 'Windows_NT'">DOTNET_VERSION_URL=file:%2F%2F$(LocalBuildInfoRoot) $(ProjectDirectory)build_projects/update-dependencies/update-dependencies$(ShellExtension) --Update</UpdateCommand>
```
[the remark this answers is not in the document]
Comment (eerhardt) [reply targets/cli.props]: Which env var?

`UpdateCommand` is still being honored here:  

https://github.com/dotnet/source-build/blob/335c5582bea0d4b1a9758e9ac777cd86182ffdf8/targets/repository.proj#L99-L101

**Comment by eerhardt** [comment]
test RHEL7.2 Release

**Comment by mmitche** [comment]
test RHEL7.2 Release

**Comment by eerhardt** [comment]
Whooo hooo!

**Pull request merged**